### ISSUE TITLE
How to tell lessc to use correct nodejs bin path

### ISSUE THREAD

**Issue by anentropic** [less/less.js#2683]
I'm trying to get Less working with the Ubuntu package version of Node.js

```
apt-get install -y nodejs
apt-get install -y npm
npm install -g less@'<2.0.0'
```

This looks like it ought to work, but it doesn't. NPM works fine but `lessc` gives the following error:

```
/usr/bin/env: node: No such file or directory
```

...the problem is that the Ubuntu package installs Node.js as `nodejs` and Less is looking for a bin called `node` (in fact it's in the shebang line at the top of `lessc`).

Is there any way to install Less so that it knows the correct Node.js path?

I realise I could symlink or alias my `nodejs -> node` or hand-edit the `lessc` file... I'd just rather use a less hacky way if one exists.

**Comment by seven-phases-max** [comment]
See #1997. I.e. instead of trying to "fix" `lessc` and every other `node` script you'll install, you should fix your node installation (either by creating a symlink in `/usr/bin/env` or by installing the proper `node` package (`apt-get install nodejs-legacy`)).

**Comment by seven-phases-max** [comment]
Closing as duplicate of #1997.

**Comment by Gesias** [comment]
I ended up here also, my issue is that a legacy project is using lessc and node to compile a few less files and lessc wanted to use `#/usr/bin/env node`  at the very start of the file. For some reason that did not work in the context even though I could run the command without problems. I just ended up changing the line in the lessc file to go straight for `#!/usr/local/bin/node` instead. Hacky? For sure, but this is running in a linux container in prod and this is just to make this part work in the OSX dev env. I expect to upgrade these packages and make this an non issue soon.
